src/main/java/kg/peaksoft/giftlistb6/dto/responses/NotificationResponse.java

```diff
@@ -0,0 +1,21 @@
+package kg.peaksoft.giftlistb6.dto.responses;
+
+import kg.peaksoft.giftlistb6.enums.NotificationType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class NotificationResponse {
+
+    private Long id;
+    private String fullName;
```

Comment:
Remove fullName and add userId, firstName, lastName
Rename createdDate to createdAt